Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/BaseListboxController.java

```diff
@@ -793,8 +793,7 @@ public void onEvent(Event evt) throws Exception {
             switch (((Integer) evt.getData())) {
               case Messagebox.YES:
                 deleteElements(mainController);
-                mainController.loadWorkspace();
-                refreshContent();
+                mainController.reloadSummariesWithOpenTreeItems(mainController.getNavigationController().getAllOpenFolderItems());
```

Comment:
There is a private refreshWorkspace() method in BaseListboxController which contains the same code as this line. Perhaps we can reuse that method in this class to avoid duplication.

Reply:
private refreshWorkspace() actually contains only same line
"mainController.reloadSummariesWithOpenTreeItems(mainController.getNavigationController().getAllOpenFolderItems());"

Reply:
OK... Using That!